### PR TITLE
Change the resource strings we use when adding a project or metadata reference.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
Here's what it looks like:

image

File: src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.MetadataSymbolReference.cs [hunk 1]
```diff
@@ -18,6 +19,19 @@ public MetadataSymbolReference(AbstractAddImportCodeFixProvider<TSimpleNameSynta
                 _reference = reference;
             }
 
+            protected override string TryGetDescription(
+                Project project, SyntaxNode node, SemanticModel semanticModel)
+            {
+                var description = base.TryGetDescription(project, node, semanticModel);
```
Comment (Pilchie) [inline src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.MetadataSymbolReference.cs]: Can you comment on why we ignore the result?

**Comment by Pilchie** [comment]
:+1:

**Pull request merged**